src/lib/constants.js

```diff
@@ -58,6 +58,7 @@ module.exports = Object.freeze({
   ),
   USERNAME_REGEX: new RegExp(/^[a-zA-Z0-9\-_.]{2,20}$/),
   PHONE_NUMBER_REGEX: new RegExp(/^[+]?[0-9]{1,3}[-\s]?[0-9]{1,3}[0-9]{4,9}$/),
+  AGE_REGEX: new RegExp(/^[0-9]+$/),
```

Comment:
Could you please change it to `AGE_REGEX: new RegExp(/^[1-9]{1}\d{1}$/),`? This one is more functional.